app/src/main/res/layout/fragment_gameboard.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/gameboard_recycler_view"
+        android:layout_width="match_parent"
```

Comment:
to work properly in a `ConstraintLayout` the height and width should be `0dp`.
Then you can set constraints to wherever you want to have your RecyclerView. I suggest to make it a percentage of the screen height. Constraining to a guideline can be used to achieve that (see `fragment_sample`)